src/main/java/datameer/awstasks/ant/ec2/AbstractEc2Task.java

```diff
@@ -51,6 +51,7 @@ private AmazonEC2 createEc2() {
         if (_region != null && !_region.trim().isEmpty()) {
             ec2Client.setRegion(Region.getRegion(Regions.valueOf(_region.toUpperCase())));
         }
+        LOG.info("connect to region " + _region);
```

Comment:
region can be null would be nice if seperate log message could be used or this one moved into the if statement